### PR TITLE
travis-worker restart cron jobs.

### PR DESCRIPTION
**Pull request by emdantrim**
This should be a nice bandage for our recent BB troubles.

- [ ] verified in staging

### CODE REVIEW

File: cookbooks/travis_worker/recipes/bluebox.rb [hunk 1]
```diff
@@ -104,3 +104,44 @@
     notifies :run, resources(:execute => 'monit-reload')
   end
 end
+
+file '/etc/default/travis-worker-restart' do
+  content "TRAVIS_WORKER_RESTART_SLEEP=#{node['travis']['worker']['restart_sleep']}"
+  owner "root"
+  group "root"
+  mode 0644
```
Comment (meatballhat) [inline cookbooks/travis_worker/recipes/bluebox.rb]: This gets rewritten every `chef-client` run, so maybe add a `not_if { File.exists?('/etc/default/travis-worker/restart') }`.

Comment (meatballhat) [inline cookbooks/travis_worker/recipes/bluebox.rb]: I dunno.  If the file gets corrupt, we'd want it re-written.  Maybe leave it like this.  nbd.

**Comment by meatballhat** [comment]
LGTM :shipit:

File: cookbooks/travis_worker/recipes/bluebox.rb (hunk 1 above)
Comment (solarce) [inline cookbooks/travis_worker/recipes/bluebox.rb]: You can do it with a grep that could verify some content

Comment (meatballhat) [inline cookbooks/travis_worker/recipes/bluebox.rb]: `not_if 'grep -E "TRAVIS_WORKER_RESTART_SLEEP=\d+" /etc/default/travis-worker-restart'` maybe?

**Pull request merged**